### PR TITLE
TSDM Update (to version 0.8)

### PR DESCRIPTION
**Pull request by remyb2canada**
#1335  and #1310

### CODE REVIEW

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 1]
```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
...
 
-This reality around the need to reduce risks and revisit the relationship stakeholders have with technology, is well explained in:
+Moving to a highly integrated, complex, and uncertain world that is Digital warrant us to move towards a more empirical cycle of trying, observing, and course correcting. The relationship between the Planning and Execution Phase becomes cyclical. It's through execution that we gain the necessary empirical evidence to inform our Planning estimates and decide on course corrections. Recent software development practices, mainly Cloud and DevOps, have permitted us to move towards this approach. IT no longer requires lengthy wait time to procure servers, to develop large code bases, and to seek large capital investments for infrastructure. Cloud has commoditized IT infrastructure enabling ESDC to rapidly develop, test, and deploy software.
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:707]: "...warrants us to". (add an s to warrant). But i would suggest changing to 'requires us to' as I feel like that is more easily understood.

And further down "..IT no longer requires lengthy wait times...' (add an s to time)

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 2]
```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
...
-9. The [House of Commons questions](https://large-government-of-canada-it-projects.github.io) from June 2016 and May 2019 regarding ongoing or planned IT projects over $1M .
-Statistics includes :
-    - Of the 94 projects that contain sufficient data to compare schedules: 9% are on schedule, 4% are ahead of schedule, and 87% are behind schedule
-    - Of the 97 projects that contain sufficient data to compare budgets: 26% are within 10% original estimates, 28% are between 10% and 50% above their original estimates, 28% are above 50% of their original estimates, and 19% are below 10% of their original estimates.
+In addition, the software world is moving away from [Monolith-types of solutions](https://medium.com/koderlabs/introduction-to-monolithic-architecture-and-microservices-architecture-b211a5955c63) (large code bases that provides many capabilities but become bottlenecks and single-points of failures in the IT ecosystem as the many IT teams needed to perform work collide).
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:709]: "...large code bases that provide" (remove the s)

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 3]
```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
...
+- [The Project Sponsor is responsible for:] 4.2.18 Establishing a project gating plan at the outset of the project, consistent with the department's framework, that [4.2.18.1] Documents the decisions that will be taken at each gate, the evidence and information required in support of the gate decisions, the criteria used to assess the evidence, and the gate governance
 
-IITB’s [BPMO Project Portfolio dashboard](http://dialogue/grp/ITPM-GPTI/PPDMLibrary/BRM-RPT-PDM-ProjectPortfolioDashboard20200217.xlsm) also tracks lite projects, branch initiatives, and A-based investments (smaller in size than the ones tracked by the EPMO).
-The February 2020 statistics show a total of 26 active projects of this category.
-69% of them are deemed healthy (green), 15% are deemed of concerns (yellow), and 15% are deemed unhealthy (red).
+The above 3 requirements from the TB Directive indicates that departments must accept and adapt to change, as well as make evidence-based decision making (like planning decisions). Such evidence can only be obtained by execution.
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:717]: Just because of the other comments, I would suggest changing the 3 to three

Also..."...as well as make evidence-based decisions" (rather than decision making)

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 4]
```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
...
-However, ESDC's relationship with technology spans beyond IITB’s influence.
+IITB has made efforts to modernize its management of technology, as is showcased in the [IITB News Kudo's Corner](http://esdc.prv/en/iitb/corporate/news/archives.shtml)[^14]  and its [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental IMIT Plan/19-20 Plans/IITB Moving Forward v2.docx)[^15] plan. However, ESDC's relationship with technology spans beyond IITB's influence.
 
-Moving ESDC towards being an agile organization requires moving towards a model that enables smaller, more frequent software deployments.
-To make more frequent software deployments requires revisiting software development and management practices as a whole.
+Moving ESDC towards being an agile organization requires moving towards a model that enables smaller, more frequent software deployments as they are enabling the organization to gain empirical evidence necessary to make evidence-based decision making.
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:721]: Same as above...I would say "...to make evidence-based decisions" or "...to do evidence-based decision making"

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 5]
```diff
@@ -839,21 +882,20 @@ Figure 2. Relationship Model between the different software elements (part 2).
...
 
-**IT Project**
+**IT-Enabled Project**
 
 A temporary endeavour undertaken to create a unique technology product, service, or result.
-The temporary nature of IT projects indicates that an IT project has a definite beginning and end.
+The temporary nature of IT-Enabled projects indicates that it has a definite beginning and end.
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:888]: Because 'projects' is plural I would change this to ...."...indicates that they have a definite...."  Or instead "..indicates that there is a definite...."

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 6]
```diff
@@ -923,6 +947,87 @@ In this strategy, it is expected to have IITB and CFOB work together for during
...
+
+**1) Standish Group study**
+
+The Standish Group, a research advisory organization that focuses on software development performance[^16], found that “of 3,555 projects from 2003 to 2012 that had labor costs of at least $10 million, only 6.4% were successful. The Standish data showed that 52% of the large projects were “challenged” meaning they were over budget, behind schedule or didn't meet user expectations. The remaining 41.4% were failures — they were either abandoned or started anew from scratch."[^17]
+
+The standish group study and results were also mentioned in Chapter 3 of the Novembre 2006 Report of the Auditor General of Canada (statements 3.5 and 3.6). See point 3 below
+
+**2) 2016 and 2019 House of Commons Questions (projects of more than $1M)**
+
+The Canadian Digital Service (CDS) [released a dataset](https://large-government-of-canada-it-projects.github.io/)[^18] collected in response to two written questions in the House of Commons, from June 2016 and May 2019. Each question asked federal government departments to report ongoing or planned IT projects over $1M.
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:962]: It wasn't CDS...It was Sean and Lucas but the were doing that work as part of a CivicTech project (so not part of their day jobs).  Would rephrase to "Thanks to an Ottawa Civic Tech project, a dataset on large government IT projects was released with responses collected from two written...."

**Review by RachelMuston — COMMENTED** [review]
This is awesome Rémy!  All but one of my comments are grammatical only. The content is excellent (I included some yay ! comments to show my enthusiasm :)).  My only non-grammatical change is the reference to the large IT project dataset..this wasn't released by CDS. It was Ottawa Civic Tech.

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 7]
```diff
@@ -38,57 +37,79 @@ _Note 2: This page will link to internal ESDC documents, which are unfortunately
...
+This includes both IITB and non-IITB stakeholders such as from CFOB, IAERMB, and SSC.
+The list of stakeholders is listed in [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the implementation of this strategy.
+
+The **Guiding Policy**, once operationalized, will target stakeholders involved when IT-enabled Projects are conceptualized and created, when IT Solutions are architected, and when IT Products are developed, delivered, operationalized, and maintained. All ESDC personnel involved in IT investment decisions are expected to adhere to this Guiding Policy.
+
+### Business Case (Strategic Context)
+
+Moving to the digital age exposes the ubiquitous nature of technology in delivering services to Canadians. As such, for ESDC to iterate on its policies and service offerings[^1], it will need to involve IT. In order to better deliver services to Canadians, we need to improve IT's responsiveness otherwise the feedback loop between policy makers and Canadians will be too long, affecting ESDC's ability to iterate on its policy making and become an agile organization.
```
[the remark this answers is not in the document]
Comment (remyb2canada) [reply _pages/en/strategy-target-solution-delivery-model.md]: reworded. It's more about IT's responsiveness. "Involving IT" meant that the traditional means of "involving IT" through advance planning and detailed requirements may not work as well in the digital world.

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 8]
```diff
@@ -38,57 +37,79 @@ _Note 2: This page will link to internal ESDC documents, which are unfortunately
...
+
+![An image showing 3 icons: Policy Makers, Technology, and Canadians. Between each icons, there's a relationship displayed as such: Policy Makers must use Technology to implement their policies, Technology is used to delivery services to Canadians, Canadians provide feedback to Policy Makers]({{ site.baseurl }}/assets/images/policy_through_technology.png)
+*Figure 1. In the Digital age, technology is between Policy Makers and Canadians*
+
+> ESDC requires **production deployments** to gain the **empirical evidence** needed for **evidence-based decision making**.
+
+ESDC is not a new department. It has accumulated a [significant amount of technical debt](http://www.oag-bvg.gc.ca/internet/English/parl_oag_201004_01_e_33714.html)[^2] over its decades that affects the responsiveness of IT.
```
[the remark this answers is not in the document]
Comment (remyb2canada) [reply _pages/en/strategy-target-solution-delivery-model.md]: reworded

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 9]
```diff
@@ -38,57 +37,79 @@ _Note 2: This page will link to internal ESDC documents, which are unfortunately
...
+ESDC's [application portfolio contains more than 500 applications](https://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/ESDC%20IT%20Plan%202019-22%20-%20FINAL.pdf)[^3], 101 of which are mission critical and depend on legacy technologies (aged custom build applications and a mainframe).
```
[the remark this answers is not in the document]
Comment (remyb2canada) [reply _pages/en/strategy-target-solution-delivery-model.md]: I've used wordings from the IT Plan. Prefer to keep it as such.

**Review by RachelMuston — APPROVED** [review]
